frontend/src/components/ShootWorkers/ManageWorkers.vue

```diff
@@ -158,12 +159,18 @@ export default {
     },
     cloudProviderKind () {
       const cloudProfile = this.cloudProfileByName(this.cloudProfileName)
-      return cloudProfile.metadata.cloudProviderKind
+      return get(cloudProfile, 'metadata.cloudProviderKind')
     },
     currentZonesNetworkConfiguration () {
       return getZonesNetworkConfiguration(this.zonesNetworkConfiguration, this.internalWorkers, this.cloudProviderKind, this.allZones.length, this.existingWorkerCIDR)
     }
   },
+  watch: {
+    currentZonesNetworkConfiguration (value) {
```

Comment:
I general we should use getter and setter and handle the dependent Data there if possible.